stories/1-base/badges.stories.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+import { storiesOf } from '@storybook/react'
+
+storiesOf('Base|Badge', module)
```

Comment:
@PRASHANTK22111993 This should be in the components group rather than base. So you'll need to move this file to the directory called `stories/4-components` and also change line 5 to 
```
storiesOf('Components|Badges', module)
```

I also changed Badge to Badges on line 5 since the majority of the other stories use the plural of the component name, ie. 'Checkboxes', 'Switches', 'Buttons', etc.